### PR TITLE
PR for Frontend GUI with backend interaction

### PR DESCRIPTION
**Pull request by achouliarop**
@callebjorkell

### CODE REVIEW

**Comment by callebjorkell** [comment]
Please merge #2 first, so that I don't need to check the same code twice.

File: src/main/java/com/tradeshift/controller/HelloWorldController.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping("/")
+public class HelloWorldController {
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ModelAndView Index() {
```
Comment (callebjorkell) [inline src/main/java/com/tradeshift/controller/HelloWorldController.java]: naming.

File: src/main/webapp/WEB-INF/jsp/index.jsp [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>
+    <title>Welcome Traveller</title>
```
Comment (callebjorkell) [inline src/main/webapp/WEB-INF/jsp/index.jsp:6]: I'm going to go ahead and assume you stole this from that travel app? :smile:

Comment (achouliarop) [inline src/main/webapp/WEB-INF/jsp/index.jsp:6]: What travel app? :D 
No it's from a game and when you open the store it says welcome traveller :P

**Comment by callebjorkell** [comment]
Could you also do the same as Kasper and post a demo video of the frontend? Quicktime can do screen captures, and you can upload the video to box or to youtube or wherever you want. Just a quick demo of the functionality.

**Comment by achouliarop** [comment]
Sure I'll do it :)

**Comment by achouliarop** [comment]
http://youtu.be/5C9L4E8GZ5c

**Comment by callebjorkell** [comment]
very nice. :+1:

**Pull request merged**